generators/client/templates/react/src/main/webapp/app/shared/layout/header/header.tsx.ejs

```diff
@@ -115,13 +115,15 @@ export class Header extends React.Component<IHeaderProps, IHeaderState> {
       <DropdownItem tag={Link} key="configuration" to="/admin/configuration"><FaList /> Configuration</DropdownItem>,
     <%_ if (databaseType !== 'cassandra') { _%>
       <DropdownItem tag={Link} key="audits" to="/admin/audits"><FaBell /> Audits</DropdownItem>,
-    <%_ } _%>
-      <DropdownItem tag={Link} key="logs" to="/admin/logs"><FaTasks /> Logs</DropdownItem>,
-      /* jhipster-needle-add-element-to-admin-menu - JHipster will add entities to the admin menu here */
-      <DropdownItem tag={Link} key="docs" to="/admin/docs"><FaBook /> API Docs</DropdownItem><% if (devDatabaseType === 'h2Disk' || devDatabaseType === 'h2Memory') { %>,
+      <%_ } _%>/* jhipster-needle-add-element-to-admin-menu - JHipster will add entities to the admin menu here */  
```

Comment:
can you put the needle in a new line, also indent the closing ejs tag with opening tag